app-shells/fish/fish-2.4.0.recipe

```diff
@@ -59,8 +59,14 @@ GLOBAL_WRITABLE_FILES="
 BUILD()
 {
 	autoreconf -vfi
-	CFLAGS=-D_BSD_SOURCE LDFLAGS="-lnetwork" runConfigure ./configure \
+
+	export CFLAGS="-D_BSD_SOURCE"
+	export CXXFLAGS=$CFLAGS
+	export LIBS="-lnetwork -lexecinfo"
```

Comment:
AC_SEARCH for socket waiting for merge in upstream : https://github.com/fish-shell/fish-shell/pull/3675
AC_Search for execinfo already in the latest git version.
So i just don't want to invest too much work in this commit right now.